media/css/pebbles/components/_global-nav.scss

```diff
@@ -515,6 +531,7 @@ html.moz-nav-open .moz-global-nav-page-mask {
 
         .download-title {
             font-weight: bold;
+            white-space: nowrap;
```

Comment:
Hmm... well both should be the same, but now I'm not sure which way to go and maybe I should remove it from the SCSS. What do you think: let the button wrap or not?